Source/DafnyDriver/Commands/CommandRegistry.cs

```diff
@@ -28,8 +28,8 @@ static class CommandRegistry {
     AddCommand(new ResolveCommand());
     AddCommand(new VerifyCommand());
     AddCommand(new RunCommand());
-    AddCommand(new BuildCommand());
```

Comment:
No more build? :' (

Reply:
What's happening here? It's the second time this repeats, and I did not touch it. I wonder what's going on in my IDE. Thanks for checking.